src/components/MarqueeSkills.tsx

```diff
@@ -0,0 +1,53 @@
+import { SKILLS } from '@/data/skills';
+import marqueeStyles from '@/styles/marquee.module.css';
+
+interface MarqueeSkillsProp {
+    direction: 'forward' | 'reverse';
+}
```

Comment:
Once we use the default value we can make the property optional. 

What will it improve?
- Now, JSX is much cleaner when you use this component. 
- Safer usage as we have default value
- Since it is not a compulsory prop, it is much easier for `consumers` to use it. 
- Testing is much easier (though we are not testing it yet. For future rerence)


```suggestion
interface MarqueeSkillsProp {
    direction?: 'forward' | 'reverse';
}
```